Composer/packages/client/src/components/CreationFlow/CreateOptions.tsx

```diff
@@ -264,78 +53,47 @@ export function CreateOptions(props: CreateOptionsProps) {
           .get<any>(`/api/projects/alias/${alias}`)
           .then((aliasRes) => {
             if (aliasRes.status === 200) {
-              navigate(`/bot/${aliasRes.data.id}`);
+              setIsOpen(true);
               return;
             }
           })
           .catch((e) => {
-            setIsOpen(true);
+            setIsOpenCreateModal(true);
           });
         return;
       }
     }
-    setIsOpen(true);
+    setIsOpenCreateModal(true);
```

Comment:
suggestion: `setIsOpen` and `setIsOpenCreateModal` are a little confusing. I would consider renaming them.

Reply:
Sure